Email-validation/script.js

```diff
@@ -0,0 +1,23 @@
+
+
+function EmailValidation (Email)
+{
+    const template= /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
please explain to me if you have been knowing what this line exactly is?

Reply:
hey Elias 
I was answered you in another issue related in Email validation.
 But I don't understand how this is , I pressed here to do pull request related in Id validation .
Did you check my new branch (Id validation ) ?!

Reply:
![image](https://user-images.githubusercontent.com/39850311/163172383-0ad44c56-27cc-4bc5-90b4-09598f2d6e63.png)